src/docs/components/gallery.html

```diff
@@ -118,22 +118,22 @@ <h3>HTML</h3>
    &#x3C;div class=&#x22;qg-image-gallery&#x22;&#x3E;
        &#x3C;ul&#x3E;
             &#x3C;li&#x3E;&#x3C;a title=&#x22;caption 1&#x22; data-fancybox=&#x22;gallery-92732&#x22; data-caption=&#x22;caption 1&#x22;
-                   href=&#x22;https://via.placeholder.com/450&#x22;&#x3E;&#x3C;img alt=&#x22;&#x22; src=&#x22;https://via.placeholder.com/150&#x22;&#x3E;&#x3C;/a&#x3E;
+                   href=&#x22;https://placehold.co/450&#x22;&#x3E;&#x3C;img alt=&#x22;&#x22; src=&#x22;https://placehold.co/150&#x22;&#x3E;&#x3C;/a&#x3E;
```

Comment:
https://www.reddit.com/r/web_design/comments/10lqgja/placeholdercom_is_no_more/

we should really be using our own inhouse assets for speed and reduced risk of another buyout.